### PR TITLE
Problem: no local gravity bridge setup instructions

### PR DESCRIPTION
**Pull request by yihuang**
Closes: #52

Solution:
- document the steps of gravity bridge setup
- change signer_set_txs to latest_signer_set_tx for simpler code in integration test

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: docs/gravity-bridge-setup-guide.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+### Cronos Testnet
+
+Either using a public cronos testnet (that have embed the gravity-bridge module), or run `start-cronos /tmp/test-cronos` to get a local Cronos testnet.
+
+Own some funds in this testnet, for the local testnet, you'll get the funds with the same private key as above.
+
+## Generate Orchestrator Keys
+
+You'll need to prepare two accounts for orchestrator, one for ethereum and one for cronos, you can also using a same private key for both accounts. Both accounts need to have some basic funds to convert the gas fees for messages relaying.
```
Comment (yihuang) [inline docs/gravity-bridge-setup-guide.md:29]:
```suggestion
You'll need to prepare two accounts for orchestrator, one for ethereum and one for cronos, you can also using a same private key for both accounts. Both accounts need to have some basic funds to cover the gas fees for messages relaying.
```

**Review by leejw51crypto — APPROVED** [review]
lgtm

File: docs/gravity-bridge-setup-guide.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+/tmp/test-cronos` to get a local Cronos testnet.
+
+You should own some funds in this testnet, for the local testnet, you'll get the funds with the same private key as
+above.
+
+## Generate Orchestrator Keys
+
+You need to prepare two accounts for the orchestrator, one for ethereum and one for cronos, you can also use a same
+private key for both accounts. You should transfer some funds to these accounts, so the orchestrator can cover the gas
+fees of messages relaying later.
```
Comment (thomas-nguy) [inline docs/gravity-bridge-setup-guide.md:36]: is there a pre-command to generate the keys?

File: docs/gravity-bridge-setup-guide.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+Use your favorite protobuf library to encode the message, and use your favorite web3 library to do the messge signing,
+for example, this is how it could be done in python:
+
+```python
+msg = DelegateKeysSignMsg(validator_address=val_addr, nonce=nonce)
+sign_bytes = eth_utils.keccak(msg.SerializeToString())
+
+acct = eth_account.Account.from_key(...)
+signed = acct.sign_message(eth_account.messages.encode_defunct(sign_bytes))
+return eth_utils.to_hex(signed.signature)
```
Comment (thomas-nguy) [inline docs/gravity-bridge-setup-guide.md:63]: as a todo, create a script for this 🙏

File: docs/gravity-bridge-setup-guide.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+- `eth_addresses` and `powers`:
+  - Query signer set by running command: `cronosd q gravity latest-signer-set-tx | jq ".signer_set.signers"`
+  - Sum up the `power` field to get `powers`
+  - Collect the `ethereum_address` field into a list to get `eth_addresses`
+
+At last, use your favorite web3 library/tool to deploy the gravity contract with the above parameters in the ethereum
+testnet, the compiled artifacts of the contract (`Gravity.json`) can be found in [gravity-bridge's
+releases](https://github.com/PeggyJV/gravity-bridge/releases).
+
+## Run Orchestrator
```
Comment (thomas-nguy) [inline docs/gravity-bridge-setup-guide.md:93]: does this also run relayer?

**Review by thomas-nguy — APPROVED** [review]
LGTM overall 
May need to ask @lezzokafka input as well

File: docs/gravity-bridge-setup-guide.md (hunk 4 above)
Comment (yihuang) [reply docs/gravity-bridge-setup-guide.md:93]: Yes, it'll relay messages in both directions.

File: docs/gravity-bridge-setup-guide.md (hunk 3 above)
Comment (yihuang) [reply docs/gravity-bridge-setup-guide.md:63]: `gorc sign-delegate-keys` cando that, I've created an issue to migrate from orchestrator cli to gorc cli later.
https://github.com/crypto-org-chain/cronos/issues/78

**Comment by tomtau** [comment]
`gorc` doesn't need to be used?

**Comment by tomtau** [comment]
https://github.com/PeggyJV/gravity-bridge/blob/490da0176a74be7d4aad17f301f303c2da2968f9/testnet/gorc_test_bootstrap.sh#L12

**Comment by yihuang** [comment]
> `gorc` doesn't need to be used?

gorc can be used to replace the orchestrator cli, gorc provides several other util functionalities, plan to try it later.

File: docs/gravity-bridge-setup-guide.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+```shell
+$ orchestrator --cosmos-phrase="{mnemonic_phrase_of_cronos_acount}" \
+    --ethereum-key={private_key_of_ethereum_account} \
+    --cosmos-grpc=http://localhost:{cronos_grpc_port} \
+    --ethereum-rpc={ethereum_web3_endpoint} \
+    --address-prefix=ethm --fees=basetcro \
```
Comment (yihuang) [inline docs/gravity-bridge-setup-guide.md:100]:
```suggestion
    --address-prefix=crc --fees=basetcro \
```

File: docs/gravity-bridge-setup-guide.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+## Run Orchestrator
+
+```shell
+$ orchestrator --cosmos-phrase="{mnemonic_phrase_of_cronos_acount}" \
+    --ethereum-key={private_key_of_ethereum_account} \
+    --cosmos-grpc=http://localhost:{cronos_grpc_port} \
+    --ethereum-rpc={ethereum_web3_endpoint} \
+    --address-prefix=crc --fees=basetcro \
+    --contract-address={gravity_contract_address} \
+    --metrics-listen 127.0.0.1:3000
```
Comment (yihuang) [inline docs/gravity-bridge-setup-guide.md:102]:
```suggestion
    --metrics-listen 127.0.0.1:3000 --hd-wallet-path="m/44'/60'/0'/0/0"
```

File: docs/gravity-bridge-setup-guide.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
+# Gravity Bridge Setup Guide
+
+## Prerequisite
+
+### Binaries
+
+- `geth`, the go-ethereum binary.
+- `cronosd`, the cronos node binary.
+- `orchestrator`, the gravity bridge orchestrator cli.
```
Comment (yihuang) [inline docs/gravity-bridge-setup-guide.md:9]:
```suggestion
- `orchestrator`, the gravity bridge orchestrator cli, build from the [crypto-org fork](https://github.com/crypto-org-chain/gravity-bridge/tree/cronos/orchestrator/orchestrator).
```

**Pull request merged**